Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -61,6 +61,15 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 PercExcl = NaN;
 MinimumtValue = NaN;
 
+if (isfield(x.modules.asl,'bMultiPLD') && x.modules.asl.bMultiPLD) || ...

```

Comment:
Would be good to define this at the start of `xASL_module_ASL`, so you don't have to do the `isfield` at each instance.